_codux/ui-kit/typography/typography.board.tsx

```diff
@@ -1,89 +1,98 @@
+                    </Variant>
+                    <Kit.Description>
+                        <b>--heading4:</b> Marcellus / 40px / 1.2
+                    </Kit.Description>
+                </Kit.Item>
+                <Kit.Item>
+                    <Variant name="Heading5">
+                        <h5 className="heading5">Heading 5</h5>
+                    </Variant>
+                    <Kit.Description>
+                        <b>--heading5:</b> Marcellus / 20px / 1.3
+                    </Kit.Description>
+                </Kit.Item>
+                <Kit.Item>
+                    <Variant name="Heading6">
+                        <h6 className="heading6">Heading 6</h6>
+                    </Variant>
+                    <Kit.Description>
+                        <b>--heading6:</b> Figtree (400) / 20px / 1.4
+                    </Kit.Description>
+                </Kit.Item>
+            </Kit.Section>
 
-            <Variant name="Paragraph1">
-                <p className="paragraph1">
-                    We ignite opportunity by setting the world in motion. 0123456789
-                </p>
-            </Variant>
-            <p className={classNames(styles.variantName, styles.headlinesSpacing)}>
-                --paragraph1: <span className={styles.fontDetails}>Figtree (400) / 20px / 1.4</span>
-            </p>
-            <Variant name="Paragraph2">
-                <p className="paragraph2">
-                    We ignite opportunity by setting the world in motion. 0123456789
-                </p>
-            </Variant>
-            <p className={classNames(styles.variantName, styles.headlinesSpacing)}>
-                --paragraph2: <span className={styles.fontDetails}>Figtree (400) / 20px / 1.4</span>
-            </p>
-            <Variant name="Paragraph3">
-                <p className="paragraph3">
-                    We ignite opportunity by setting the world in motion. 0123456789
-                </p>
-            </Variant>
-            <p className={classNames(styles.variantName, styles.headlinesSpacing)}>
-                --paragraph3: <span className={styles.fontDetails}>Figtree (400) / 20px / 1.4</span>
-            </p>
-        </div>
+            <Kit.Section title="Paragraph">
+                <Kit.Item>
+                    <Variant name="Paragraph1">
+                        <div className="paragraph1">
+                            We ignite opportunity by setting the world in motion. 0123456789
+                        </div>
+                    </Variant>
+                    <Kit.Description>
+                        <b>--paragraph1:</b> Figtree (400) / 16px / 1.5
+                    </Kit.Description>
+                </Kit.Item>
+                <Kit.Item>
+                    <Variant name="Paragraph2">
+                        <div className="paragraph2">
+                            We ignite opportunity by setting the world in motion. 0123456789
```

Comment:
Can we remove this `0123456789`?

Reply:
It's by design to demonstrate how text styling looks for numbers.